en/docs/concepts/oidc/id-token.md

```diff
@@ -0,0 +1,61 @@
+# Introduction to ID Token
+
+### ID Token
+ID Token is a security token that contains claims about the authentication of an end-user by an *Authorization Server*
+which is introduced with [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html#IDToken).
+This token represents as a JWT (Json Web Kit) and contains user claims to identify who the user is and it contains a unique 
+identifier that is represented in the ‘sub’ claim. The ID Token is issued from the authorization server and the audience
+of the ID Token is the client application. The client application will validate the ID Token based on the signature value
+and can interpret the value of the ID Token. To secure the ID Tokens the token should be signed and it can be also encrypted.
+
+There are two main usage of the id token.
+
+1. To exchange with the access token.
+
+2. To obtain the user information and pass it to external applications if required.
```

Comment:
There are two main uses for the ID Token. 

1. It can be exchanged with the access token. 

2. It can obtain user information and pass it to external applications if required.